### PR TITLE
chapter 8 basic login

### PR DESCRIPTION
**Pull request by tuanhnt1712**

### CODE REVIEW

**Comment by tuanhnt1712** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
```
Comment (dieunb) [inline app/controllers/sessions_controller.rb]:
```
def create
  # bad
  user = User.find_by email: params[:session][:email].downcase
  if user && user.authenticate(params[:session][:password])
    # something
  else
    #something
  end
end

def create
  # good
  user = User.find_by email: params[:session][:email].downcase

  if user && user.authenticate(params[:session][:password])
    # something
  else
    #something
  end
end
```

**Comment by dieunb** [comment]
commented

**Comment by tuanhnt1712** [comment]
ready

**Comment by dieunb** [comment]
commented-(merged) đi cưng (yaoming)

**Pull request merged**